### DISCUSSION TITLE
Update to Prisma Schema

### DISCUSSION THREAD

**Discussion by mdwiltfong** [mdwiltfong/secret-santa#17] [Announcements]
As of this [PR](https://github.com/mdwiltfong/secret-santa/commit/d8460420e0f0f0cd25dc4567e2238ca92e790ba3) being merged, the prisma schema has been updated. As a result you will have to update your Prisma Schema locally. I have added notes on how to do this in the [README.md](https://github.com/mdwiltfong/secret-santa#readme) file

**Comment by gracekishino** [comment]
Thanks for this! I could update the Prisma schema and seed the data in the DB locally.
